src/pages/Signup/terms.js

```diff
@@ -0,0 +1,33 @@
+function Modal({ setModalOpen, setCheckAll }) {
+  const closeModal = () => {
+    setModalOpen(false);
+  };
+
+  return (
+    <div className="termsWrap">
+      <span>모달창입니다.</span>
+      <button onClick={closeModal}>X</button>
+      <div>회원 약관</div>
+      <div>
+        <input type="checkbox" />
+        <span>[필수] 개인정보 제3자 제공 동의</span>
+      </div>
+      <br></br>
```

Comment:
```suggestion
      <br />
```
JSX문법중에 self closing 잊지마세요~

Reply:
넵 반영하였습니다!